### PR TITLE
Deprecate unused social strategies

### PR DESCRIPTION
**Pull request by eessex**
Removes functionality and related libs for oauth via Twitter and LinkedIn.

Addresses https://artsyproduct.atlassian.net/browse/GROW-794

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,12 +32,6 @@ app.use artsyPassport
   # ----------------
   FACEBOOK_ID: # Facebook app ID
   FACEBOOK_SECRET: # Facebook app secret
-  TWITTER_KEY: # Twitter consumer key
-  TWITTER_SECRET: # Twitter consumer secret
-  TWITTER_KEY: # Twitter consumer key
-  TWITTER_SECRET: # Twitter consumer secret
```
Comment (joeyAghion) [inline README.md]: Were these duplicated in error?

File: lib/index.coffee [hunk 2]
```diff
@@ -10,7 +9,7 @@ setupApp = require './app/index'
 setupPassport = require './passport/index'
 artsyXapp = require 'artsy-xapp'
 
-module.exports = (options) =>
+module.exports = (options) ->
```
Comment (joeyAghion) [inline lib/index.coffee]: Subtle... is this a fix or something else?

File: README.md (hunk 1 above)
Comment (eessex) [reply README.md]: Probably, but we won't need any of them anymore

File: lib/index.coffee (hunk 2 above)
Comment (eessex) [reply lib/index.coffee]: Minor, my vscode coffee linting complains because we don't need `this` for the function to complete.

Comment (eessex) [reply lib/index.coffee]: This is also why I added braces in a number of places.

**Pull request merged**